content/en/platform/corda/4.12/enterprise/node-upgrade-notes.md

```diff
@@ -306,7 +306,7 @@ node requires.
 
 
 {{< important >}}
-Corda 4 requires Java 8u171 or any higher Java 8 patch level. Java 9+ is not currently supported.
+Corda 4 requires Java 17.
```

Comment:
### for this line:

> Corda 4 requires Java 17.0.9 or any higher Java 17 patch level. Java 18+ is not currently supported.

### for this file:

Please can you also add the changes to this file from this PR:
https://github.com/corda/corda-docs-portal/pull/2176/files#diff-c7f6e94ae2a65c463894d51e4b7675b67328f83cb40e252fb67969143f640448


There are quite a lot of changes to this file needed